README.md

```diff
@@ -7,96 +7,59 @@
 [![Go Reference](https://pkg.go.dev/badge/github.com/bridgecrewio/yor.svg)](https://pkg.go.dev/github.com/bridgecrewio/yor)
  
-Cloud service providers allow users to assign metadata to their cloud resources in the form
-of tags. Each tag is a simple label consisting of a customer-defined key and a value
-that can make it easier to manage, search for, and filter resources. Although there are no
-inherent types of tags, they enable customers to categorize resources by purpose, owner,
-environment, or other criteria.
+Yor is an open-source tool that helps add informative and consistent tags across infrastructure-as-code frameworks such as Terraform, CloudFormation, and Serverless.
 
-Tags can be used for security, cost allocation, automation, console organization, access control, and operations. 
-
-Yor is an open-source tool that helps to manage tags in a consistent manner across infrastructure as code frameworks (Terraform, Cloudformation, and Serverless Framework) .
-By auto-tagging in IaC you will be able to trace any cloud resource from code to cloud. 
-Yor enables version-controlled owner assignment and resource tracing based git history. It also can extend tag enforcement logic by loading external tagging logic into the CI/CD pipeline. 
+Yor is built to run as a [GitHub Action](https://github.com/bridgecrewio/yor-action) that hydrates IaC code with consistent tagging logics. It can also run as a pre-commit hook and a standalone CLI.
 
 ## Features
-* Simple tagging - tag a directory IaC resource by user input
-* Git tagging - Yor collects data from [git-blame](https://git-scm.com/docs/git-blame) and enables mapping individual resources to specific commits and users.
-* ```yor_trace``` tag enables full attribution between build time and run time resources.
+* Tracing: ```yor_trace``` tag enables simple attribution between an IaC resource block and a running cloud resource.
+* Change management: git-based tags automatically add org, repo, commit and modifyer details on every resource block.  
+* Custom taggers: user-defined tagging logics can be added to run using Yor.
+* Skips: inline annotations enable developers to excluse paths that should not be tagged.
 
 ## Demo
-> Generate an git tags on infrastructure as code
-![](docs/yor_git_tags.gif)
-
-> Track resource last modifying user from code to cloud
+### Attributing a resource to an owner
 ![](docs/yor_owner.gif)
 
-> Track resource identifer from code to cloud
+### Change management tags
+![](docs/yor_git_tags.gif)
+
+### Trace IaC code to cloud resource
 ![](docs/yor_trace.gif)
 
-> Track resource code block file cloud to code
+### Trace cloud resource to IaC code
 ![](docs/yor_file.gif)
 
 ## **Table of contents**
 
 - [Getting Started](#getting-started)
 - [Disclaimer](#disclaimer)
 - [Support](#support)
-
-More Docs:
 - [Customizing Yor](CUSTOMIZE.md)
 
 ## Getting Started
 
 ### Installation
+GitHub Action
```

Comment:
@guyeisenkot We need to specify 2 things with this:
1. The command does not commit the changes
2. We need the checkout beforehand to get us the git history of the repo. That will be done by setting the `fetch-depth: 0` attribute as described in the action docs